audio for bird and rocket

linchen2508 opened the pull request:
Add sound for bird and rocket engine in both scenes
(Becareful your ears)

linchen2508 committed:
more audio

Add sound for bird and rocket engine in both scenes
(Becareful your ears)

linchen2508 committed:
Merge branch 'main' into audio-and-timer

linchen2508 committed:
Fix conflict error

Update from main to fix conflict error

Aether700 reviewed (changes requested):
Cannot hear any sound effects from the rocket (was using the porta potty in the canyon scene)
also, the bird sound is very aggressing. Would it be possible to either find a new sound (I can help with that if needed) or at least lower the volume a bit?

linchen2508 committed:
Change volume

linchen2508 committed:
Different bird sounds

linchen2508 committed:
Change bird sound

Aether700 reviewed (changes requested):
Volume for the main theme, the rockets and the birds is too low please raise slightly.

Also, please delete all the un-used bird sound files

linchen2508 committed:
Remove redundant files + raise volume

linchen2508 committed:
Add volume slider

Add volume slider for music and sound effect ( used text mesh pro so there are many stuff imported)

Aether700 reviewed (changes requested):
Please put the slider on the main menu scene or add a pause menu to pause the game and then adjust the sound from there.

Also, please add a slider for the theme music as the theme music is way too loud on my computer

linchen2508 committed:
Fix 2 slider disappearing

linchen2508 committed:
Add a pause menu

Press Esc to turn on/off pause menu

Aether700 commented on Assets/Scripts/UI/PauseMenu/PauseMenu.cs:
```diff
@@ -0,0 +1,47 @@
+
+public class PauseMenu : MonoBehaviour
+{
+    public static bool GameIsPaused = false;
```
please do not make this public and make it private with a getter

Aether700 commented on Assets/Scripts/UI/PauseMenu/PauseMenu.cs:
```diff
@@ -0,0 +1,47 @@
+    public void Resume()
+    {
+        pauseMenuUI.SetActive(false);
+        Time.timeScale = 1f;
```
more for your information but it's better practice to use events to control when the game is paused. For this project using time.timescale is more than fine

Aether700 reviewed (changes requested):
good job just a few changes to make

linchen2508 committed:
change volume slider range + make variable private
